gio/Cargo.toml

```diff
@@ -6,7 +6,7 @@ authors = ["The gtk-rs Project Developers"]
 keywords = ["glib", "gio", "gtk-rs", "gnome"]
 readme = "README.md"
 documentation = "https://gtk-rs.org/gtk-rs-core/stable/latest/docs/gio/"
-version = "0.18.1"
+version = "0.18.2"
```

Comment:
Please change version in all Cargo.toml, but only upload/publish the ones that have functional changes